src/core/expr/fexpr.cc

```diff
@@ -669,6 +671,69 @@ DECLARE_METHOD(&PyFExpr::first)
     ->docs(doc_first);
 
 
+static const char* doc_as_type =
+R"(as_type()
+--
+
+Equivalent to :func:`dt.as_type(self)`.
+)";
+
+oobj PyFExpr::as_type(const XArgs& args) {
+  auto as_typeFn = oobj::import("datatable", "as_type");
+  oobj n = args[0].to_oobj();
+  return as_typeFn.call({this, n});
+}
+
+
+DECLARE_METHOD(&PyFExpr::as_type)
+    ->name("as_type")
+    ->docs(doc_as_type)
+    ->arg_names({"n"})
+    ->n_positional_or_keyword_args(1);
+
+
+static const char* doc_cov =
+R"(cov()
+--
+
+Equivalent to :func:`dt.cov(self)`.
+)";
+
+oobj PyFExpr::cov(const XArgs& args) {
+  auto covFn = oobj::import("datatable", "cov");
+  oobj n = args[0].to_oobj();
+  return covFn.call({this, n});
+}
```

Comment:
Frankly, I'm not sure whether `cov()` and `corr()` should be converted into methods at all. For example, when I write `corr(f.A, f.B)` it's clear that this computes the correlation coefficient between columns A and B. However, form `f.A.corr(f.B)` is asymmetric in its arguments and thus very confusing. What do you think?